apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -46,48 +50,107 @@ function StakeForm({
 
     const [formatted, symbol] = useFormatCoin(coinBalance, coinType);
 
+    const [coinDecimals] = useCoinDecimals(coinType);
+
+    const [gasBudgetEstimation] = useFormatCoin(
+        DEFAULT_GAS_BUDGET_FOR_STAKE,
+        coinType
+    );
+
+    const coinBalanceMinusGas =
```

Comment:
Why not just do `coinBalance - DEFAULT_GAS_BUDGET_FOR_STAKE`?

Reply:
Also similar as above, that for now is fine and not sure if will change in the future but we can only do this if coinType is SUI. We can ignore gas and keep coinBalance is coinType is not SUI

Reply:
This is done

Reply:
```suggestion
    const coinBalanceMinusGas = coinBalance - BigInt(coinType === SUI_TYPE_ARG ? DEFAULT_GAS_BUDGET_FOR_STAKE : 0);
```

Reply:
done